lookout/style/typos/nn_prediction.py

```diff
@@ -0,0 +1,110 @@
+from typing import Iterable, Tuple
+
+from gensim.models.fasttext import FastText
+import keras
+from keras.layers import Dense
+from keras.models import Sequential
+import numpy
+import pandas
+
+from lookout.style.typos.utils import collect_embeddings, CORRECT_TOKEN_COLUMN, TYPO_COLUMN
+
+
+def get_features(fasttext: FastText, typos: Iterable[str]) -> numpy.ndarray:
+    return numpy.concatenate((collect_embeddings(fasttext, typos),
+                             numpy.ones((len(typos), 1))), axis=1)
+
+
+def get_target(fasttext: FastText, identifiers: Iterable[str]) -> numpy.ndarray:
+    return collect_embeddings(fasttext, identifiers)
+
+
+def create_model(input_len: int, output_len: int) -> keras.models.Sequential:
+    model = Sequential()
+    model.add(Dense(units=256, activation='relu', input_dim=input_len))
```

Comment:
quotes here and in other places should be double quotes

Reply:
Thanks, forgot to check. Any idea what can be wrong with tensorflow install in travis?